test/pca.jl

```diff
@@ -156,3 +158,114 @@ M = fit(PCA, X2, pratio=0.85)
 M = fit(PCA, X2, pratio=0.85f0)
 # Float64 input, specified Float32 pratio
 M = fit(PCA, X, pratio=0.85f0)
+
+
+
+
+
+
+## prepare training data
```

Comment:
Can you explain what these tests check compared with those above? Also, no need for more than two empty lines.